released/packages/coq-actuary/coq-actuary.2.4/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "palmskog@gmail.com"
+
+homepage: "https://github.com/Yosuke-Ito-345/Actuary"
+dev-repo: "git+https://github.com/Yosuke-Ito-345/Actuary.git"
+bug-reports: "https://github.com/Yosuke-Ito-345/Actuary/issues"
+license: "MIT"
+
+synopsis: "Formalization of actuarial mathematics in Coq"
+description: """
+This package formalizes basic actuarial mathematics using Coq and
+the Mathematical Components and Coquelicot libraries. This includes
+the theory of interest, life tables, annuities, and reserves."""
+
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+depends: [
+  "coq" {>= "8.13"}
+  "coq-mathcomp-ssreflect" {>= "1.12" & < "1.14~"}
```

Comment:
@Yosuke-Ito-345 is the 2.4 release of Actuary compatible with MathComp 1.15.0? If so, you should increase the upper bound for `coq-mathcomp-ssreflect` as per my suggestion:
```suggestion
  "coq-mathcomp-ssreflect" {>= "1.12" & < "1.16~"}
```
On the other hand, if Actuary 2.4 is only compatible with MathComp 1.12.0 and 1.13.0, I can merge the pull request as it is now.

Reply:
@palmskog I'm sorry. `coq-actuary-2.4` is compatible with `mathcomp-1.14.0`, but incompatible with `mathcomp-1.15.0`. So I

Reply:
yes

Reply:
Thank you for the reply. I will make the revised pull request later.